sdk/spring/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/core/CosmosOperations.java

```diff
@@ -212,6 +226,15 @@ public interface CosmosOperations {
      */
     <T> void deleteEntity(String containerName, T entity);
 
+    /**
+     * Delete using a list of entities with bulk
+     *
+     * @param <T> type class of domain type
+     * @param containerName the container name
+     * @param cosmosItemOperations the Flux of the CosmosItemOperation's to delete
+     */
+    <T> void deleteEntities(String containerName, Flux<CosmosItemOperation> cosmosItemOperations);
```

Comment:
why do we have a `Flux<>` in CosmosOperations? Shouldn't this be Iterable?

Reply:
Updating.